bigbone/src/main/kotlin/social/bigbone/api/method/ListMethods.kt

```diff
@@ -20,4 +24,118 @@ class ListMethods(private val client: MastodonClient) {
             method = MastodonClient.Method.GET
         )
     }
+
+    /**
+     * Fetch the list with the given ID. Used for verifying the title of a list, and which replies to show within that list.
+     * @param listId The ID of the List in the database.
+     * @see <a href="https://docs.joinmastodon.org/methods/lists/#get-one">Mastodon API documentation: methods/lists/#get-one</a>
+     */
+    fun getList(listId: String): MastodonRequest<MastodonList> {
+        return client.getMastodonRequest(
+            endpoint = "api/v1/lists/$listId",
+            method = MastodonClient.Method.GET
+        )
+    }
+
+    /**
+     * Create a new list.
+     * @param title The title of the list to be created.
+     * @param repliesPolicy One of [MastodonList.RepliesPolicy], defaults to [MastodonList.RepliesPolicy.List].
+     * @see <a href="https://docs.joinmastodon.org/methods/lists/#create">Mastodon API documentation: methods/lists/#create</a>
+     */
+    @JvmOverloads
+    fun createList(
+        title: String,
+        repliesPolicy: MastodonList.RepliesPolicy = MastodonList.RepliesPolicy.List
+    ): MastodonRequest<MastodonList> {
+        return client.getMastodonRequest(
+            endpoint = "api/v1/lists",
+            method = MastodonClient.Method.POST,
+            parameters = Parameters().apply {
+                append("title", title)
+                append("replies_policy", repliesPolicy.value)
+            }
+        )
+    }
+
+    /**
+     * Change the title of a list, or which replies to show.
+     * @param listId The ID of the List in the database.
+     * @param title The new title of the list to be updated.
+     * @param repliesPolicy One of [MastodonList.RepliesPolicy].
+     * @see <a href="https://docs.joinmastodon.org/methods/lists/#update">Mastodon API documentation: methods/lists/#update</a>
+     */
+    fun updateList(
+        listId: String,
+        title: String,
+        repliesPolicy: MastodonList.RepliesPolicy
```

Comment:
We should set a default value here and add JvmOverloads (I have commented the same in the RxMethod).

Reply:
If we allow a default value here, updating a list might lead to an unintended change in replies policy, if a list with a different replies policy gets updated with the default value.

Mastodon documentation claims that this has a server-side default value (implying that it should be optional), but I feel that would just lead to the same problem.

Reply:
Good point. Fine for me.